versions.txt

```diff
@@ -34,8 +34,8 @@ autoinstrumentation-go=v0.19.0-alpha
 
 # Represents the current release of Apache HTTPD instrumentation.
 # Should match autoinstrumentation/apache-httpd/version.txt
-autoinstrumentation-apache-httpd=1.0.4
```

Comment:
Why are you bumping this? There's a PR for it [here](https://github.com/open-telemetry/opentelemetry-operator/pull/3488) and it looks like it's not clear what is actually supported.

Reply:
argh ill undo...